crates/compute/src/memory.rs

```diff
@@ -72,3 +72,25 @@ pub trait ComputeMemory<F> {
 		Self::split_at_mut(borrowed, mid)
 	}
 }
+
+pub struct FSliceWithDegree<'a, F, Mem: ComputeMemory<F>> {
+	pub slice: Mem::FSlice<'a>,
+	pub degree: u8,
```

Comment:
Per discussion with @benediamond, this should either be named `log_degree` or `tower_level`. `tower_level` seems clearer given the corresponding constant associated with `TowerField`.